lms/djangoapps/branding/api.py

```diff
@@ -367,18 +367,20 @@ def get_base_url(is_secure):
     return _absolute_url(is_secure=is_secure, url_path="")
 
 
-def get_logo_url():
+def get_logo_url(is_secure=True):
```

Comment:
yes, by default urls should be secure unless specified otherwise, other methods in this file follow the same convention.